README.md

```diff
@@ -62,6 +62,21 @@ The frontend includes components, logic, and API interaction for:
   - [frontend/README](./frontend/README.md)
   - [backend/README](./backend/README.md)
 
+## Technical requirements
+
+This product was built to be capable of running in the following environment, where candidates are tested.
+
+- Internet Explorer 10 or later
+- JavaScript enabled
+- Secure Socket Layer (SSL) encryption enabled
+- Screen resolution must be set to a minimum of 1024 x 768
```

Comment:
Just for documentation purposes, we'll need to update our status page with this minimum screen resolution, since it is now set to a minimum of  800x600:

![image](https://user-images.githubusercontent.com/23021242/61301404-cf458300-a7b1-11e9-9b38-3e58c8d12c4f.png)

Reply:
Good call - updated: https://github.com/code-for-canada/project-thundercat/pull/480